integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownOptionsEnvDomainCluster.java

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer.Alphanumeric;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+/**
+ * Simple JUnit test file used for testing Operator.
+ *
+ * <p>This test is used for applying Shutdown Properties at Domain, Cluster, Env level.
+ */
+@TestMethodOrder(Alphanumeric.class)
+public class ItPodsShutdownOptionsEnvDomainCluster extends ShutdownOptionsBase {
+
+  private static Operator operator1 = null;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception If initializing of properties failed.
```

Comment:
if - no capital
either "initializing properties" or "initialization of properties"

Reply:
done